libethereum/BlockChain.cpp

```diff
@@ -298,7 +299,7 @@ unsigned BlockChain::open(fs::path const& _path, WithExisting _we)
         auto const l = m_extrasDB->lookup(db::Slice("best"));
         m_lastBlockHash = h256(l, h256::FromBinary);
     }
-    catch (db::FailedLookupInDB const& /* ex */)
```

Comment:
Changes like these might lead to more exceptions being unhandled, but if it's low-level IO error or DB corruption, we can't do much but exit anyway I guess

Reply:
See comment above: Errors should be treated as fatal, reported nicely and exit.